### PR TITLE
[FIX] l10n_co: fix es translations

### PR DESCRIPTION
**Pull request by aliyatastemirova**
At the moment translations are only available if a user chooses es_CO specifically. This commit changes it to es, making it available regardless of which Spanish version is chosen by the user.

task-3230384

---
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

**Comment by jco-odoo** [comment]
@aliyatastemirova Is not es_ES Spanish from Spain instead of generic Spanish?

File: addons/l10n_co/data/l10n_co_chart_data.xml [hunk 1]
```diff
@@ -9,7 +9,7 @@
         <field name="cash_account_code_prefix">1105</field>
         <field name="transfer_account_code_prefix">1115</field>
         <field name="country_id" ref="base.co"/>
-        <field name="spoken_languages" eval="'en_US;es_CO'"/>
+        <field name="spoken_languages" eval="'en_US;es_CO;es_ES'"/>
```
Comment (jco-odoo) [inline addons/l10n_co/data/l10n_co_chart_data.xml:12]: maybe this change is not necessary.

Comment (aliyatastemirova) [reply addons/l10n_co/data/l10n_co_chart_data.xml:12]: es_ES seems to be the generic Spanish (also Spanish from Spain). I think this option impacts what languages are loaded during module installation, so could still be necessary?

**Comment by jco-odoo** [comment]
@robodoo r+

**Pull request closed without merging**